packages/dso-toolkit/reference/render/dropdown-menu--dropdown-default-button-open-checkable.html

```diff
@@ -1,22 +1,22 @@
-<dso-dropdown-menu checkable=""><button type="button" id="dropdown_default_button_open_checkable" aria-haspopup="true" aria-expanded="false" slot="toggle" class="btn btn-default">
+<dso-dropdown-menu checkable=""><button type="button" id="dropdown_default_button_open_checkable" aria-haspopup="menu" aria-expanded="false" slot="toggle" class="btn btn-default">
     <span>Actie</span>
   </button>
   <div hidden="">
     <div class="dso-dropdown-options">
```

Comment:
hier mist dus nog `role="menu"` maar dat gaat vanzelf goed komen als we dat gaan programmeren in `packages\core\src\components\dropdown-menu\dropdown-menu.tsx`, zie boven